bluemira/codes/openmc/output.py

```diff
@@ -448,13 +451,19 @@ def from_openmc_csg_result(cls, result: OpenMCResult):
         """
         source = "OpenMC CSG"
 
+        auxiliary_power = (
+            result.total_power
+            - result.blanket_power
+            - result.divertor_power
+            - result.vessel_power
+        )
```

Comment:
P_aux is usually defined for power deposited in "auxiliary" systems, which is typically used for in-vessel components that are not the blankets or divertor (e.g. HCD systems, port plugs, whatever).

Here we don't have any auxiliary systems modelled as it is axisymmetric.